src/pages/Jobs/MyRecentJobs/index.jsx

```diff
@@ -16,20 +17,114 @@
  51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 */
 
-import React from "react";
+import React, { useEffect, useState } from "react";
 
 // Title
 import Title from "components/Title";
 
+// Required functions for API and Error handling
+import { getAllJobApi } from "api/jobs";
+import { handleError } from "shared/helper";
+import { initialMessage } from "constants/constants";
+import messages from "constants/messages";
+import { Alert } from "components/Widgets";
+import { MDBDataTable } from "mdbreact";
+import { getUserSelf } from "services/users";
+
 const MyRecentJobs = () => {
+  // Setting the browse data to the table
+  const [jobsDataList, setJobsDataList] = useState();
+
+  // State Variables for handling Error Boundaries
+  const [message, setMessage] = useState(initialMessage);
+  const [showMessage, setShowMessage] = useState(false);
+
+  function Dateformatter(date) {
+    const options = {
+      weekday: "long",
+      year: "numeric",
+      month: "long",
+      day: "numeric",
+    };
+    const d = new Date(date);
+    return d.toLocaleDateString("en-US", options);
+  }
+
+  useEffect(() => {
+    const arr = [];
+    setMessage({
+      type: "success",
+      text: messages.loading,
+    });
+    setShowMessage(true);
+
+    getAllJobApi()
+      .then((res) => {
+        // Checking the id of user with the userId from the jobs list
+        const uid = getUserSelf();
+        uid.then(function getUid(result) {
+          const usrId = result.id;
+          let k = 0;
+          for (let i = 0; i < res.length; i++) {
+            // Formatting the date from time stamp to readable date
+            res[i].queueDate = Dateformatter(res[i].queueDate);
+            if (res[i].userId === `${usrId}`) {
+              arr[k] = res[i];
+              k += 1;
+            }
+          }
+          setJobsDataList(arr);
+        });
+
+        setShowMessage(false);
+      })
+      .catch((error) => {
+        handleError(error, setMessage);
+      });
+  }, []);
+
+  // Data formatted for the data-table with respective coloumns
+  const data = {
+    columns: [
+      {
+        label: "Job Name ",
+        field: "name",
+        sort: "asc",
+        width: 150,
+      },
+      {
+        label: "Queue Date",
+        field: `queueDate`,
+        sort: "asc",
+        width: 270,
+      },
+      {
+        label: "Status",
+        field: "status",
+        sort: "asc",
+        width: 200,
+      },
+    ],
+    rows: jobsDataList,
+  };
+
   return (
     <>
-      <Title title="My Recent Jobs" />
-      <div className="main-container my-3">
+      <Title title="Show Jobs" />
```

Comment:
Any reason to change the title here?

Reply:
No particular reason reverting it to the original title!
Resolved

Reply:
I was not asking for a revert :-)
Just wanted to understand the reason behind it.